### PR TITLE
Add a warning when node is syncing

### LINKED ISSUE #5376: Be explicit with warning that busy syncing has outdated info
While it's syncing, the UI should block with a button "I know it will show out-of-date information, let me use the wallet anyway" and a checkbox "never show this again"

### PR DESCRIPTION
**Pull request by ngotchac**
Closes #5376 

image

### CODE REVIEW

**Comment by jacogr** [comment]
Looks very out of place - could we at least align it with the Connect dialog (closest I think to look & feel & placement)

**Comment by ngotchac** [comment]
You mean having the modal at the top?

File: js/src/views/SyncWarning/syncWarning.js [hunk 1]
```diff
@@ -0,0 +1,130 @@
...
+            <FormattedMessage
+              id='syncWarning.message.line1'
+              defaultMessage={ `
+                Your Parity node is still syncing to the chain.
+              ` }
+            />
+            <FormattedMessage
+              id='syncWarning.message.line2'
+              defaultMessage={ `
+                Some of the shown informations might be out-of-date.
```
Comment (rphmeier) [inline js/src/views/SyncWarning/syncWarning.js]: nit: should just be "information"

**Comment by ngotchac** [comment]
Updated

**Pull request merged**